src/univariate/continuous/cosine.jl

```diff
@@ -33,32 +45,34 @@ mode(d::Cosine) = d.μ
 
 var(d::Cosine) = d.σ^2 * 0.13069096604865779  # 0.130... = 1/3 - 2 / π^2
 
-skewness(d::Cosine) = 0.0
+skewness{T<:Real}(d::Cosine{T}) = zero(T)
 
-kurtosis(d::Cosine) = -0.59376287559828102362
+kurtosis{T<:Real}(d::Cosine{T}) = -0.59376287559828102362*one(T)
```

Comment:
Wouldn't be correct for `BigFloat`s. On the other hand, my suggestion fails for integers. Generic coding can be tricky.